### DISCUSSION TITLE
gatsby-plugin-typography - Error while following Gatsby tutorial part three

### DISCUSSION THREAD

**Discussion by delroh** [gatsbyjs/gatsby#31192] [Help] [answered]
## Description

Following the tutorial on [part three](https://www.gatsbyjs.com/docs/tutorial/part-three/), while installing `gatsby-plugin-typography`, npm is unable to resolve dependency tree.

### Steps to reproduce

The same steps in the tutorial: 
1. Create a new Gatsby site
```shell
gatsby new tutorial-part-three https://github.com/gatsbyjs/gatsby-starter-hello-world
cd tutorial-part-three
```
2. Install and configure `gatsby-plugin-typography`
```shell
npm install gatsby-plugin-typography react-typography typography typography-theme-fairy-gates
```
This results in the error `unable to resolve dependency tree`. See the screenshot:

![Screenshot 2021-04-30 022557](https://user-images.githubusercontent.com/3700414/116656884-7382c480-a95b-11eb-989d-56314b507e8b.png)

### Expected result

`npm install gatsby-plugin-typography react-typography typography typography-theme-fairy-gates` should install successfully.

### Actual result

npm doesn't install anything because there's a dependency conflict.

### Environment

```
  System:
    OS: Windows 10 10.0.19042
    CPU: (12) x64 Intel(R) Core(TM) i7-10750H CPU @ 2.60GHz
  Binaries:
    Node: 14.16.1 - C:\Program Files\nodejs\node.EXE
    npm: 7.11.1 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Spartan (44.19041.906.0), Chromium (90.0.818.49)
  npmPackages:
    gatsby: ^3.4.0 => 3.4.0 
  npmGlobalPackages:
    gatsby-cli: 3.4.0
```

**Comment by LekoArts** [comment, marked as answer]
Hi!

You can install the dependencies with the `--legacy-peer-deps` flag, e.g. `npm install --legacy-peer-deps gatsby-plugin-typography`

**Reply by delroh** [reply]
Thank you! Saw that option in the npm docs but wasn't sure if it was safe to use and wouldn't break anything with dependencies. Anyways, I skipped installing it and continued the tutorial, since that plugin is just for styling purposes and I was still able to understand the concept the tutorial is conveying without actually installing it.
